zgrab_schema.py

```diff
@@ -800,29 +805,23 @@
                     "compression":String(),
                 }),
             }),
-            "dh_key_exchange": SubRecord({
-                "params": SubRecord({
-                    "prime": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "generator": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "client_public": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "client_private": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
+            "key_exchange": SubRecord({
+                "ed25519sha256_params": SubRecord({
```

Comment:
Thank you for the clarification. Addressed in 43bcf80bc9473fc1f3ed08bcd2d605f70dc82b1e